fluentgen/src/main/java/com/azure/autorest/fluent/mapper/FluentMapper.java

```diff
@@ -67,9 +87,96 @@ public FluentClient map(CodeModel codeModel, Client client) {
+                        FluentCollectionMethod collectionMethod = collectionMethodOptional.get();
+                        FluentCollectionMethodExample collectionMethodExample = ExampleParser.parseMethodExample(resourceCollection, collectionMethod, example);
+                        FluentExampleTemplate.ExampleMethod exampleMethod = fluentExampleTemplate.generateExampleMethod(collectionMethodExample);
+                        setExampleStepFeatures(result, testCase, fluentStep, collectionMethodExample, exampleMethod);
+                    } else {
+                        // find resourceCreate
+                        Optional<ResourceCreate> createMethod = findResourceCreate(resourceCollection, operation);
+                        if (createMethod.isPresent()) {
+                            ResourceCreate create = createMethod.get();
+                            FluentResourceCreateExample createExample = ExampleParser.parseResourceCreate(resourceCollection, create, example);
+                            FluentExampleTemplate.ExampleMethod exampleMethod = fluentExampleTemplate.generateExampleMethod(createExample);
+                            setExampleStepFeatures(result, testCase, fluentStep, createExample, exampleMethod);
+                        } else {
+                            // find resourceUpdate
+                            Optional<ResourceUpdate> updateMethod = resourceCollection.getResourceUpdates().stream().filter(rc -> FluentUtils.exampleIsUpdate(rc.getMethodName()) && rc.getMethodName().equalsIgnoreCase(operation)).findFirst();
+                            if (updateMethod.isPresent()) {
+                                ResourceUpdate update = updateMethod.get();
+                                FluentResourceUpdateExample updateExample = ExampleParser.parseResourceUpdate(resourceCollection, update, example);
+                                if (updateExample == null) {
+                                    continue parseLiveTests;
+                                }
+                                FluentExampleTemplate.ExampleMethod exampleMethod = fluentExampleTemplate.generateExampleMethod(updateExample);
+                                setExampleStepFeatures(result, testCase, fluentStep, updateExample, exampleMethod);
+                            }
+                        }
+                    }
+                    testCase.getSteps().add(fluentStep);
+                    result.getHelperFeatures().addAll(testCase.getHelperFeatures());
+                } else {
+                    continue parseLiveTests;
+                }
+            }
+
+            result.getTestCases().add(testCase);
+
+        }
+
+        return result;
+    }
+
+    private void setExampleStepFeatures(FluentLiveTests result, FluentLiveTestCase testCase, FluentExampleLiveTestStep fluentStep, com.azure.autorest.fluent.model.clientmodel.examplemodel.FluentExample createExample, FluentExampleTemplate.ExampleMethod exampleMethod) {
```

Comment:
the `createExample` seems not always a create example?

Reply:
Renamed.